modules/openy_gc_auth/modules/openy_gc_auth_daxko_sso/src/Controller/DaxkoLinkController.php

```diff
@@ -133,22 +134,30 @@ public function backlink(Request $request) {
     // Based on user token, get logged user data.
     $userData = $this->daxkoClient->getMyInfo($token);
 
-    $userDetails = $this->daxkoClient->getRequest('members/' . $userData->member_id);
-    // Check if this user is an active client.
-    if ($userDetails->active) {
-      // Create drupal user if it doesn't exist and login it.
-      $name = $userDetails->name->first_name . ' ' . $userDetails->name->last_name;
-      $email = $userDetails->emails[0]->email;
-
-      // Authorize user (register, login, log, etc).
-      $this->gcUserAuthorizer->authorizeUser($name, $email);
-
-      return new RedirectResponse($this->configFactory->get('openy_gated_content.settings')->get('virtual_y_url'));
-    }
-    else {
-      // Redirect back to Virual Y login page.
-      return new RedirectResponse($this->configFactory->get('openy_gated_content.settings')->get('virtual_y_login_url'));
+    if ($userData) {
+      $userUnitData = $this->daxkoClient->getRequest('/units/' . $userData->member_unit_id);
+      $userDetails = $this->daxkoClient->getRequest('members/' . $userData->member_id);
```

Comment:
let's stay consistent with having or not leading slash. ( Lines 138-139 )

Reply:
done